feat(one-shot): add falcon prepare command (#3879)
hashgraph/solo: docs/site/content/en/templates/advanced-deployments.template.md

## Falcon Deployment

Falcon deployment provides fine-grained control over all network components through a YAML configuration file. This is
ideal for CI/CD pipelines, automated testing, and complex deployment scenarios.

> **Note**: The `--deployment` flag must **not** be used inside a values file for the `solo one-shot falcon` command.
> The deployment name is auto-generated by the command and cannot be overridden through the values file. Specifying
> `--deployment` in the values file will cause the deployment to fail.

### Preparing a Falcon Configuration

> Instead of writing a values file by hand, use the interactive wizard to generate one:

Reviewer: I think this implies a bit that writing it is discouraged, maybe explaining that both are valid is better

```suggestion
You can either create a values file manually or use the interactive wizard to generate one:
```

> **Note**: The `--deployment` flag must **not** be used inside a values file for the `solo one-shot falcon` command. The deployment name is auto-generated by the command and cannot be overridden through the values file. Specifying `--deployment` in the values file will cause the deployment to fail.

### Preparing a Falcon Configuration

Instead of writing a values file by hand, use the interactive wizard to generate one:

```bash
solo one-shot falcon prepare
```

The wizard prompts for component toggles (mirror node, explorer, relay), consensus node count, component versions, ingress, storage type, developer options, and port forwarding. All prompts have sensible defaults so you can press Enter to accept them.

> To generate a values file with all defaults (no prompts):

Reviewer: is it explained anywhere? also you are using relative path, which will behave different when using solo outside of the root directory
